codec/std/msgs.go

```diff
@@ -5,12 +5,12 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/cosmos/cosmos-sdk/x/evidence"
 	eviexported "github.com/cosmos/cosmos-sdk/x/evidence/exported"
-	"github.com/cosmos/cosmos-sdk/x/gov"
+	gov "github.com/cosmos/cosmos-sdk/x/gov/types"
 )
 
 var (
 	_ eviexported.MsgSubmitEvidence = MsgSubmitEvidence{}
-	_ gov.MsgSubmitProposalI        = MsgSubmitProposal{}
+	_ gov.MsgSubmitProposalI        = &MsgSubmitProposal{}
```

Comment:
Do we need to do the same thing for `MsgSubmitEvidence`?

Reply:
Should it be `MsgSubmitEvidenceI`? Also maybe we do this in another PR to keep the scope small?